pkgs/by-name/la/lact/package.nix

```diff
@@ -63,11 +66,19 @@ rustPlatform.buildRustPackage rec {
   '';
 
   postFixup = lib.optionalString stdenv.targetPlatform.isElf ''
-    patchelf $out/bin/.lact-wrapped --add-needed libvulkan.so --add-rpath ${
-      lib.makeLibraryPath [ vulkan-loader ]
```

Comment:
Please keep the logic this way; only adding needed for the .so name, not the exact path to the .so.

AFAIUI, this allows overriding the .so used using LD_LIBRARY_PATH.